src/commands/validate.ts

```diff
@@ -41,20 +38,26 @@ export const validate = command<Options>({
         description: "Prints all inventory.",
         default: false,
       })
+      .option("framework", {
+        type: "string",
+        choices: ["pss-restricted", "pss-baseline", "nsa"] as const,
+        alias: "f",
```

Comment:
Not strongly opinionated, but `f` is a nice and commonly used shortcut for `file`. Not sure if we ever need it but something to consider. I guess we can always remove it, it's not the worst breaking change.

Reply:
Agreed, changed it to `fw` instead